code/modules/vehicle/janicart.dm

```diff
@@ -5,10 +5,16 @@
 	icon_state = "pussywagon"
 	key_type = /obj/item/key/janitor
 	var/obj/item/storage/bag/trash/mybag
+	var/datum/action/floor_buffer/F
```

Comment:
Regardless of the QDEL_NULL this will likely fail to GC. Can you make the reference soft and just create a new one whenever it's inserted
See #19212 and use the same logic

Reply:
Like this?

Reply:
Yep, should work a-ok, although I'd probably change the f var to something like "floorbuffer_action"